### PR TITLE
Docs fix for kubectl proxy configuration

### PR DESCRIPTION
**Pull request by xurizaemon**
Follow-up to https://github.com/kubernetes/website/pull/32245

The `proxy-url` setting is per cluster in `~/.kube/config`, not a top-level value.

At least, using a top-level entry didn't work for me when I tried it just now! (kubectl v1.23.5)

I use the config setting for all clusters I access.

### CODE REVIEW

**Comment by reylejano** [comment]
Hi @kubernetes/sig-cli-kubectl-maintainers , please take a look at this PR and give a technical review

File: content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 1]
```diff
@@ -150,25 +150,24 @@ are stored absolutely.
...
   name: development
 
 users:
 - name: developer
 
 contexts:
 - context:
-  name: development
-
+   name: development
```
Comment (Sea-n) [inline content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:170]: Is this intended to use 3 spaces?
```suggestion
  name: development
```

**Review by Sea-n — COMMENTED** [review]
Found a nitpick, please take a look.

**Comment by zacharysarah** [comment]
@xurizaemon 👋🏻 Please feel free to `/reopen` when you're ready to respond to feedback.

/close

**Comment by xurizaemon** [comment]
Rebased & fixed the extra space. Thanks!

File: content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 1 above)
Comment (xurizaemon) [reply content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: fixed

**Comment by zacharysarah** [comment]
LGTM from a docs perspective.

@kubernetes/sig-cli-kubectl-maintainers 👋🏻 Feel free to follow with any corrections needed.

/lgtm
/approve

**Pull request merged**